.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "docs/notebooks/mit_iquhack"]
```

Comment:
Please add to README.md (line 25) before pip install . to do:

 git submodule update --init

otherwise this repo is not loaded

Same change in docs/introduction.rst (somewhere around line 33)

Reply:
You are indeed right, this would be needed to initialize submodules. At the time I didn't want to modify documentation without a request by the quantum inspire team, but it seems appropriate now!